Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -566,3 +575,23 @@ body[dir=rtl] [dir=ltr] .go-to-arrow {
         --checkerboard-dark-square: hsl(0, 0%, 5%);
     }
 }
+
+/* Keep this media query in sync with Main.js */
+@media (max-width: 600px) {
+    #content {
+        grid-template-columns: auto 1fr;
+        grid-template-rows: 1fr auto;
+
+        grid-template-areas:
+            "navigation-area browser-area"
+            "details-area details-area";
+    }
+
+    body[dir="rtl"] #content {
+        grid-template-columns: 1fr auto;
+
+        grid-template-areas:
+            "browser-area navigation-area"
+            "details-area details-area";
+    }
+}
```

Comment:
Very neat 👍 😄 

I didn't know Web Inspector didn't already handle RTL layout.
@dcrousso was that the case or am I missing something?